src/components/cylc/GraphNode.vue

```diff
@@ -77,23 +60,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           scale(${ (index === 0) ? mostRecentJobScale : '1' })
         `"
       >
-        <symbol
-          :id="`${nodeID}-${index}`"
+        <job
+          :svg="true"
+          :status="job.node.state"
           viewBox="0 0 100 100"
-          :class="`job_theme--${jobTheme}`"
```

Comment:
Job theme no longer needs to be specified because we are using the job icon in the document natively rather than embedding it as a symbol.